Brewlet/AppDelegate.swift

```diff
@@ -136,20 +136,22 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             packageItem.submenu?.removeAllItems()
             
             var iconName = ""
-            self.updatesAvailable = n_lines > 0
-            if self.updatesAvailable {
+            let previousOutdatedPackageCount = self.outdatedPackageCount
+            self.outdatedPackageCount = n_lines
+            if self.outdatedPackageCount > 0 {
                 statusItem.title = "\(n_lines) Outdated Packages"
                 iconName = "BrewletIcon-Color"
                 updateItem.title = "Upgrade"
                 packageItem.isHidden = false
-                self.updatesAvailable = true
```

Comment:
Previously, this `self.updatesAvailable = true` was set within the `if self.updatesAvailable = true`. Was that intentional? Same for line 152 inside of the `else` statement.

Reply:
Not intentional, needed removing!